indico/htdocs/js/indico/RegistrationForm/tpls/fields/number.tpl.html

```diff
@@ -14,7 +14,7 @@
         {{ 'Field is mandatory' | i18n }}
     </span>
     <span ng-show="validationStarted && nestedForm.$invalid && nestedForm.$error.min " class="error-message-tag">
-        {{ 'The value must be at least' | i18n }} {{field.values.minValue}}
+        {{ 'The value must be at least' | i18n }} {{field.minValue}}
```

Comment:
Yeah, you were faster than me ;) Because I was looking for something exactly like that ;)